Btchap/target.yml

```diff
@@ -259,6 +259,9 @@ targets:
 
     # Share extension
     - path: ../RiotShareExtension/Shared
+      excludes:
+      - "**/ShareRoomsRoomCell.*"
+      - "**/ShareRoomsDiscussionCell.*"
```

Comment:
see my comment below - I think we can delete these "ShareRooms...Cell" source code